public-dashboard-client/src/testUtils.js

```diff
@@ -20,3 +44,26 @@ export { customRender as render };
 
 // provide original render method as a fallback if needed
 export { render as renderUnwrapped };
+
+// retrieve a data fixture from spotlight-api
+export function getDataFixture(filename) {
```

Comment:
Oh thats interesting. Is this fixture data manually generated or part of the nightly build and randomized? Just want to keep our eye on when/how this fixture data will become out of sync with live data and what that might mean for the tests.

Reply:
we have just been manually generating them when a new file is added to the API. I think it mostly doesn't matter if the values drift a bit, but if the shape drifts and causes tests to fail then that's a good signal that we need to update our fixtures anyway?

The one exception to this I can think of is the time-series metrics (population, supervision success rates), which have some baked-in assumptions about the data being relative to the current date. There is already a test for some of that functionality (a utility function that handles missing data) that mocks the system date to create different test conditions, so probably we would just have to do some more of that if we wanted to use those fixtures for something.